frontend/src/pages/DailySurveys/index.tsx

```diff
@@ -25,16 +28,85 @@ const validateMessages = {
 
 export default (): React.ReactNode => {
   
-  const onFinish = (values: Object) => {
-    console.log(values);
+  const onFinish = async (values: {journal: string, sleep: {count: number, rate: number}}) => {
+    
+    // Display success message
+    success();
+
+    // Request Sentiment API
+    const sentiment = await getSentiment([values.journal]);
+
+    // Constructing Database API body
+    const date = moment().format("YYYY-MM-DD");
+    const name = "Kevin";
+    const dataInput = {
+      "Name": name,
+      "Timestamp": date,
+      "Message": values.journal,
+      "Sentiment": (sentiment == "positive"? 1 : 0),
```

Comment:
Sentiment can be positive, neutral, or negative from the model. Do we want to make it binary?

Reply:
Didn't know that, will notice Benny as well

Reply:
Ill make it into string from the db side